ood-portal-generator/lib/ood_portal_generator/dex.rb

```diff
@@ -104,7 +104,8 @@ def tls_key
     end
 
     def issuer
-      "#{protocol}#{servername}:#{port}"
+      host = @config.fetch(:issuer, servername)
```

Comment:
I think if you are making `issuer` the config option, it should replace the whole value so should be something like this:

```
@config.fetch(:issuer, "#{protocol}#{host}:#{port}")
```

The the value for issuer would be the entire like `https://....` part of address.  Otherwise if you just want to specify servername, maybe make that the option and not issuer.